scripts/sync_data.py

```diff
@@ -114,6 +130,7 @@ def as_dict(self):
             "bestInCategory": self.best_in_category,
             "summary": self.summary,
             "amazonProductID": self.amazon_product_id,
+            "amazonImageURL": self.amazon_image_url,
```

Comment:
Remove this from the `as_dict` because we don't want to export this to the JSON, it unnecessarily increases the file size and memory footprint of the JSON, since this can actually be derived.